src/components/pageLayout/__tests__/__snapshots__/pageLayout.test.js.snap

```diff
@@ -23,31 +23,14 @@ exports[`PageLayout Component should render header and section children: multipl
 <PageLayout>
   <PageHeader
     key=".1"
+    title={null}
   >
     <PageHeader>
       <section
         className="pf-l-page-header pf-c-page-header pf-l-page__main-section pf-c-page__main-section pf-m-light"
         widget-type="InsightsPageHeader"
       >
-        <PageHeaderTitle
-          title="lorem"
-        >
-          <Title
-            className=""
-            headingLevel="h1"
-            size="2xl"
-            widget-type="InsightsPageHeaderTitle"
-          >
-            <h1
-              className="pf-c-title pf-m-2xl"
-              widget-type="InsightsPageHeaderTitle"
-            >
-               
-              lorem
-               
-            </h1>
-          </Title>
-        </PageHeaderTitle>
```

Comment:
#### Required:
It's a little worrisome when a huge chunk of a snapshot is removed including some HTML tags? i.e. the H1... 

I was expecting to continue to see test snapshot in addition to one confirming the ability to add an optional "sub heading"/children prop. Once you add the "isRequired" back hopefully this will be restored

Reply:
This was expected to be missing here because the title is no longer being provided in these tests.  I added a test for the PageHeader itself to account for when the title is present.  This doesn't really matter though because you want the title to be required anyway, just wanted to clarify that I had already assessed this "huge chunk" missing.